modules/ROOT/pages/deprecations-additions-removals-compatibility.adoc

```diff
@@ -3307,7 +3307,18 @@ label:restricted[]
 size()
 ----
 a|
-No longer works for paths. Only works for strings, lists and pattern expressions. See xref:functions/scalar.adoc#functions-size[size()] for more details.
+Only works for strings, lists and pattern comprehensions.
+For versions above 5.0 use a `COUNT` expression:
+[source, cypher, role="noheader"]
+----
+RETURN COUNT { (a)-[]->(b) }
+----
+For versions below 5.0 use a pattern comprehension:
+[source, cypher, role="noheader"]
+----
+RETURN size([ (a)-[]->(b) \| a ])
+----
+See xref:functions/scalar.adoc#functions-size[size()] and xref:syntax/expressions.adoc#count-subqueries[Count Expressions] for more details.
```

Comment:
If you want to match the title of the section you are linking to, it should be 'COUNT subqueries' rather than 'Count Expressions'

Reply:
Yeah, I prefer Count Expression :P But it would be less confusing :) can change it!